docs(metrics): add additional serviceMonitor helm variables (#945)
longhorn/website: content/docs/1.6.3/references/helm-values.md

| privateRegistry.registryUrl | URL of a private registry. When unspecified, Longhorn uses the default system registry. |
| privateRegistry.registryUser | User account used for authenticating with a private registry. |

### Metrics Settings

| Key | Type | Default | Description |
|-----|------|---------|-------------|

> | metrics.serviceMonitor.additionalLabels | object | `{}` | Setting that adds additional labels to the Prometheus ServiceMonitor. |
> | metrics.serviceMonitor.annotations | object | `{}` | Setting that adds annotations to the Prometheus ServiceMonitor. |

Reviewer: ```suggestion
| metrics.serviceMonitor.additionalLabels | `{}` | Additional labels for the Prometheus ServiceMonitor resource. |
| metrics.serviceMonitor.annotations | `{}` | Annotations for the Prometheus ServiceMonitor resource. |
```

Reviewer: For brevity, we directly describe the main object and include "setting" only when the supported values are true/false.

Author: Note that we cannot remove the `object` type from the table, as it is generated by helm-doc and is present in all tables on this page.

Reviewer: Apologies, that was a mistake on my end. I don't know why those cells were removed.

| metrics.serviceMonitor.enabled | bool | `false` | Setting that allows the creation of a Prometheus ServiceMonitor resource for Longhorn Manager components. |
| metrics.serviceMonitor.interval | string | `""` | Interval at which Prometheus scrapes the metrics from the target. |

> | metrics.serviceMonitor.metricRelabelings | list | `[]` | Configures the relabeling rules to apply to the samples before ingestion. See Prometheus operator documentation for format. |
> | metrics.serviceMonitor.relabelings | list | `[]` | Configures the relabeling rules to apply the target’s metadata labels. See Prometheus operator documentation for format. |

Reviewer: ```suggestion
| metrics.serviceMonitor.metricRelabelings | list | `[]` | Configures the relabeling rules to apply to the samples before ingestion. See the [Prometheus Operator documentation](https://prometheus-operator.dev/docs/getting-started/introduction/) for formatting details. |
| metrics.serviceMonitor.relabelings | list | `[]` | Configures the relabeling rules to apply the target’s metadata labels. See the [Prometheus Operator documentation](https://prometheus-operator.dev/docs/getting-started/introduction/) for formatting details. |
```

Reviewer: I linked to the intro because I couldn't find the page with formatting info.

Author: Added a more direct link.
